docs/postman/automated-test/BPDM Automated Test.postman_collection.json

```diff
@@ -0,0 +1,1112 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{Pool-Host}}/api/catena/identifier-types",
+					"host": [
+						"{{Pool-Host}}"
+					],
+					"path": [
+						"api",
+						"catena",
+						"identifier-types"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Create Metadata Add Identifier Type",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200 or 409\", function () {",
+							"    // Get the response status code",
+							"    var statusCode = pm.response.code;",
+							"",
+							"    // Check if the status code is either 200 or 409",
+							"    pm.expect(statusCode).to.be.oneOf([200, 409]);",
+							"});",
+							"pm.test(\"Body matches string\", function () {",
+							"    var responseBody = pm.response.text();",
+							"    var customAddId = pm.collectionVariables.get(\"customAddId\");",
+							"    pm.expect(responseBody).to.include(customAddId);",
+							"});",
+							""
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"var customAddId = \"CUSTOM_ADD_ID_\" + Math.random().toString(36).substring(2, 12);",
+							"pm.collectionVariables.set('customAddId', customAddId);",
+							""
+						],
+						"type": "text/javascript"
+					}
+				}
+			],
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n  \"technicalKey\": \"{{customAddId}}\",\n  \"businessPartnerType\": \"ADDRESS\",\n  \"name\": \"Custom Identifier Type of Test Automation\",\n  \"details\": [\n    {\n      \"country\": \"DE\",\n      \"mandatory\": true\n    }\n  ]\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{Pool-Host}}/api/catena/identifier-types",
+					"host": [
+						"{{Pool-Host}}"
+					],
+					"path": [
+						"api",
+						"catena",
+						"identifier-types"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Query LE Identifier Type",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"pm.test(\"Status code is 200\", function () {",
+							"    pm.response.to.have.status(200);",
+							"});"
+						],
+						"type": "text/javascript"
+					}
+				},
+				{
+					"listen": "prerequest",
+					"script": {
+						"exec": [
+							"var customLeIdValue = pm.collectionVariables.get(\"customLeId\");",
+							"",
+							"// Construct the request URL with query parameters",
+							"var requestUrl = `${pm.variables.get(\"Pool-Host\")}/api/catena/identifier-types?businessPartnerType=LEGAL_ENTITY&technicalKey=${customLeIdValue}`;",
```

Comment:
Why setting the request URL here? It is already defined in the collection

Reply:
Removed :)